plugins/source/snyk/client/spec.go

```diff
@@ -0,0 +1,40 @@
+package client
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pavel-snyk/snyk-sdk-go/snyk"
+)
+
+type Spec struct {
+	// APIKey can be set via spec or via "SNYK_API_KEY" environment variable
+	APIKey string `json:"api_key,omitempty"`
+
+	// Organizations is a list of organizations to fetch information from.
+	// By default will fetch from all organizations available for user.
+	Organizations []string `json:"organizations,omitempty"`
+
+	// EndpointURL is optional parameter to override the API URL for snyk.Client.
+	EndpointURL string `json:"endpoint_url,omitempty"`
+}
+
+const (
+	EnvSnykApiKey = "SNYK_API_KEY"
+)
+
+func (s *Spec) getClient(version string) (*snyk.Client, error) {
+	if len(s.APIKey) == 0 {
```

Comment:
```suggestion
	if s.APIKey == "" {
```

Checking for empty string using len is a bit controversial

Reply:
well, len is built into the underlying struct, so it's even faster that comparing to empty string IMO